src/jenkins/CreateIntegTestMarkDownTable.groovy

```diff
@@ -32,11 +32,11 @@ class CreateIntegTestMarkDownTable {
         def additionalInformation = """
 \nCheck out test report manifest linked above for steps to reproduce, cluster and integration test failure logs. For additional information checkout the [wiki](https://github.com/opensearch-project/opensearch-build/wiki/Testing-the-Distribution) and [OpenSearch Metrics Dashboard](https://metrics.opensearch.org/_dashboards/app/dashboards#/view/21aad140-49f6-11ef-bbdd-39a9b324a5aa).
 """
-        if(!releaseOwner.isEmpty()) {
+        if(releaseOwner != null && !releaseOwner.isEmpty()) {
```

Comment:
better to create a method `boolean isNullOrEmpty(String str) { return (str == null || str.allWhitespace || str.isEmpty()) }` and use it for null checks.